src/bin/cargo-flamegraph.rs

```diff
@@ -299,8 +303,18 @@ fn main() {
 
     flamegraph::generate_flamegraph_for_workload(
         Workload::Command(workload),
-        &flamegraph_filename,
+        &flamegraph_filename.clone(),
```

Comment:
That function takes any `AsRef<Path>` implementor, so the previous code should have been fine

```suggestion
        &flamegraph_filename,
```